utility/eslint/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@mapstore/eslint-config-mapstore",
+  "version": "1.0.1",
+  "private": false,
+  "description": "ESLint Configuration for MapStore project",
+  "author": "GeoSolutions S.a.s.",
+  "license": "ISC",
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "mapstore"
+  ],
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/geosolutions-it/MapStore2.git",
+    "directory": "utility/eslint"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
```

Comment:
should this become a dev dependency? so it will not end in the final production bundle, right?

or become peerDependencies, and add these to ms2 dev dependencies?
i'm not sure which is the best

Reply:
Yes, they are dependencies for the eslint-configuration. 
Will be downloaded as devDependencies from MapStore. If I put them as devDependencies, it means they are dependencies for development of eslint package, so they couldn't be downloaded by mapstore.